src/managers/tls.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
```

Comment:
**question** I really have a **big question** here... How much have you copied and pasted from Kafka/ZooKeeper here and in `events`?

If we have copied and pasted 100%, that would make my day. If not, also for my understanding, I believe it would be very very beneficial to segregate the general bits and the customizations needed for Kyuubi. I would then put the general class in `src/common` and here we import from common and add the custom bits. The reason why I would do this is because in this way, ideally we would already have a "proposed" implementation for the DP helpers without blocking the work here. As soon as the proposed implementation get accepted in the DP helpers, we can then remove the module in common, and replace here the import from `src.common` to `dataplatformhelpers` . Of course this is *out-of-scope* and completely optional for this PR (unless the split is a no-brainer and easy to achieve), but I wanted to start to float this idea, get your thoughts and possibly implement the split in a future PR